### ISSUE TITLE
Sword for leaves

### ISSUE THREAD

**Issue by capymachi** [Ryhon0/SimpleAutoswitch#3]
When I have a hoe and a sword on my hotbar, it defaults to the sword for leaves. It seems to instantly break leaves, but only client side

**Comment by Ryhon0** [comment]
That sounds like you have a client-server mod for faster sword leaf breaking without the server also having it. It's not something I can fix. You can either remove that mod or get a hoe which mines faster or just as fast and have it in your main hand or in a lower slot than the sword

**Comment by capymachi** [comment]
Apparently it was the vines starting the issue, here's a video with just your mod and quilted fabric API enabled:
https://www.youtube.com/watch?v=D4cSBbQlAB8
The same thing happens with an axe on my hotbar, axe would be preferred for vines.

Edit to avoid more notifications than necessary: In the video it does start with leaves, which is super confusing to me. In my further testing it's been super inconsistent, sometimes it works just fine for both vines and leaves, with the sword in the lowest slot. 

Regardless of if you can fix the issue, I really appreciate the mod, even if I have to take my sword off my hotbar sometimes :)

**Comment by Ryhon0** [comment]
Strange. Just tried it on a pure Fabric and Quilt install and it doesn't happen. Does this also happen when you're in a singleplayer world? Does it also happen when you're sneaking/have the mod disabled in the settings?

**Comment by capymachi** [comment]
I was able to replicate it in a singeplayer world earlier, now I can't replicate it almost at all, even with the same order of tools in the hotbar. Occasionally I have to stop mining for a moment when switching from leaves to logs, or the log will keep reappearing - but nothing like the original video I posted. Very strange - I had already restarted the game multiple times before that video. Sneaking/disabling the mod had the game working as usual, no switching. 

If I notice it happening again I'll try to do some more troubleshooting. I'll close the issue until then.
